contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol

```diff
@@ -4,17 +4,29 @@ pragma solidity ^0.8.0;
 import {FxBaseChildTunnel} from "../../tunnel/FxBaseChildTunnel.sol";
 import {Create2} from "../../lib/Create2.sol";
 import {Ownable} from "../../lib/Ownable.sol";
-import {FxERC20} from "../../tokens/FxERC20.sol";
+import {FxMintableERC20} from "../../tokens/FxMintableERC20.sol";
```

Comment:
Minor optimisation: Interface can be used instead of the Contract

Reply:
We're using public getters, ie `name`, `decimals` & `symbols` at [L104](https://github.com/DhairyaSethi/fx-portal-contracts/blob/mintable-erc20-fix-rebase/contracts/examples/mintable-erc20-transfer/FxMintableERC20ChildTunnel.sol#L104), can add them to the mintable ERC20 interface and standardise this across mintable erc721/1155 child tunnel examples as well. What say?